tests/integration/test_backup_restore_on_cluster/test_concurrency.py

```diff
@@ -121,10 +121,12 @@ def test_concurrent_backups_on_same_node():
 
     ids_list = "[" + ", ".join([f"'{id}'" for id in ids]) + "]"
 
+    # Wait until all the concurrent BACKUP commands have finished.
     assert_eq_with_retry(
         node0,
         f"SELECT status FROM system.backups WHERE status == 'CREATING_BACKUP' AND id IN {ids_list}",
         "",
+        retry_count=100,
```

Comment:
The test waited before for `20*0.5 == 10` seconds which wasn't enough.
Now it'll be waiting for `100*0.5 == 50` seconds which seems more appropriate.